### PR TITLE
aws-s3-mb: add page

### PR DESCRIPTION
**Pull request by shermanhui**
This PR addresses #8406 

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

Screen Shot 2022-10-20 at 20 07 35

### CODE REVIEW

File: pages/common/aws-s3-mb.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# aws s3 mb
+
+> Create an S3 bucket.
```
Comment (navarroaxel) [inline pages/common/aws-s3-mb.md:3]:
```suggestion
> Create S3 buckets.
```

File: pages/common/aws-s3-mb.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+> More information: <https://awscli.amazonaws.com/v2/documentation/api/latest/reference/s3/mb.html>.
+
+- Create an S3 bucket:
+
+`aws s3 cp {{file_name}} s3://{{example_bucket_name}}/{{file_name}}.txt`
```
Comment (navarroaxel) [inline pages/common/aws-s3-mb.md:8]:
```suggestion
`aws s3 mb s3://{{bucket_name}}`
```

File: pages/common/aws-s3-mb.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+- Create an S3 bucket in a specific region:
+
+`aws s3 mb s3://{{example_bucket_name}} --region {{region_value}}`
```
Comment (navarroaxel) [inline pages/common/aws-s3-mb.md:12]:
```suggestion
`aws s3 mb s3://{{bucket_name}} --region {{region}}`
```

**Review by navarroaxel — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**